osi_object.proto

```diff
@@ -512,6 +512,65 @@ message MovingObject
         TYPE_ANIMAL = 4;
     }
 
+    //
+    // \brief Additional pedestrian data in case the moving object is a human.
+    //
+    // This is an extension to the \c MovingObject with additional information 
+    // describing a pedestrian in more detail.
+    // 
+    message PedestrianData {
+
+        // Body height of the pedestrian  
+        // 
+        // Unit: m
+        //
+        optional double height = 1;
```

Comment:
We have that information already from Bounding Box information.
So it can be left out and then Skeleton information itself is enough and could be included directly to sth. like pedestrian_attributes